### PR TITLE
Rework the template README

### PR DESCRIPTION
**Pull request by Victoremepunto**

### CODE REVIEW

File: {{cookiecutter.project_name}}/README.md [hunk 1]
```diff
@@ -1,65 +1,108 @@
...
-**Optional for deploying to ephemeral environments:** There are recipes provided to deploy the application using [bonfire](https://pypi.org/project/crc-bonfire)
-so make sure the command is available in your PATH to run the deployment commands.
+## Installing the application
 
-## Getting started
+Most of the recipes require a Python virtual environment to be active. This is to avoid installing dependencies to your
+local user's namespace.
 
-### Creating your project
+To create and activate a virtual environment you can use the `create_venv` recipe:
```
Comment (mjholder) [inline {{cookiecutter.project_name}}/README.md]: the recipe is `venv_create` you have it the other way around

File: {{cookiecutter.project_name}}/README.md [hunk 2]
```diff
@@ -102,36 +174,46 @@ bonfire namespace reserve
...
 Make sure you're logged in against Quay using the `login` command of either `podman` or `docker` with write
 permissions against the target image repository.
 
-```
+```shell
+# podman
+podman login quay.io
+
+# docker
 podman login quay.io
```
Comment (mjholder) [inline {{cookiecutter.project_name}}/README.md]: I think this should be `docker login quay.io`

**Review by mjholder — CHANGES_REQUESTED** [review]
Looks great overall. Just a couple nit-picky things

File: {{cookiecutter.project_name}}/README.md (hunk 2 above)
Comment (Victoremepunto) [reply {{cookiecutter.project_name}}/README.md]: whops ... thank you, fixed!

File: {{cookiecutter.project_name}}/README.md (hunk 1 above)
Comment (Victoremepunto) [reply {{cookiecutter.project_name}}/README.md]: nice catch ! fixed!

**Review by mjholder — APPROVED** [review]
LGTM

**Pull request merged**